playbooks/roles/deploy-osh/tasks/main.yml

```diff
@@ -130,8 +130,25 @@
     - horizon
     - run
 
+- name: Install openstack client
+  package:
+    name: python-openstackclient
```

Comment:
could you convert that to a list with two packages?
- python-openstackclient
- python-heatclient

As the latter has a dependency on the former, not the other way around, IIRC.

Reply:
heatclient certainly does not depend on openstackclient, its the other way, aka the way jiri did it.

Reply:
that's interesting, I didn't know openstackclient bundled everything in this packaging. I double checked and it's the case indeed.

Reply:
It is not bundled in one package, it's brought by dependency